src/envoy/http/authn/filter_context.cc

```diff
@@ -47,6 +47,18 @@ void FilterContext::setOriginResult(const Payload* payload) {
   // it's ok just to check jwt payload.
   if (payload != nullptr && payload->has_jwt()) {
     *result_.mutable_origin() = payload->jwt();
+    if (payload->jwt().user().size() > 0) {
```

Comment:
Are you sure these debug prints are needed?

Reply:
These debug logs are removed.